docs/source/using/addons.adoc

```diff
@@ -180,6 +180,21 @@ Multiple variables need to be comma separated.
 # Required-Vars: PROJECT_USER
 ----
 
+Default values for variables can be achieved using the _Var-Defaults_ metadata header.
+_Var-Defaults_ need to be specified in the format of `<key>=<value>`.
+You can specify multiple default values by separating each with a comma.
+
+NOTE: The `=` character is reserved for determining which element is a key or value and cannot be used as part of either.
+
+----
+# Name: acme
+# Description: ACME add-on
+# Required-Vars: PROJECT_USER
+# Var-Defaults: PROJECT_USER=john
+----
+
+NOTE: Values specified for variables in _Var-Defaults_ will be overridden by values specified with the `--addon-env` flag or `minishift config set addon-env` command.
```

Comment:
Variable values specified via the command line using the `--addon-env` have precedence over _Var-Defaults_